lnrpc/routerrpc/router_server.go

```diff
@@ -80,6 +81,10 @@ var (
 			Entity: "offchain",
 			Action: "read",
 		}},
+		"/routerrpc.Router/SubscribeHtlcEvents": {{
+			Entity: "offchain",
```

Comment:
It isn't (or won't be) just offchain?

Reply:
We can add another entity when on chain is added.